src/components/MarketMySharesForm/index.js

```diff
@@ -89,9 +95,16 @@ class MarketMySharesForm extends Component {
     this.setState({ extendedSellId: shareId === this.state.extendedSellId ? undefined : shareId })
```

Comment:
If we are updating the state based on a previous value of itself, you have to do it with a function, otherwise you can not guarantee the state will be updated in order. Something like:
```
this.setState((state) => {extendedSellId : shareId === this.state.extendedSellId ? undefined : shareId})
```